### ISSUE TITLE
2019-12-01 tc012 M1SS/tc014 M2SS when FLGACCEPTTAU is included in PARAMETERLIST the values are not strictly 0 or 1 but some are NA

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#76]
Values for FLGACCEPTTAU are intended to be either 0 or 1 (FALSE/TRUE). However with tc012 (also verified to occur with tc014) there are records with FLGACCEPTTAU also with "NA" values. This may be because there are multiple dosing intervals with varying sampling designs.

```
> r$FLGACCEPTTAU
 [1] "1" NA  NA  "1" NA  "1" "1" "1" NA  "1" NA  "1" NA  NA  NA  NA  NA  "1" "1" NA  NA  "1" NA  NA  "1" NA  NA  NA  NA  "1" "1" "1" "1" "1" "1" "1"
```

**Comment by rudraya** [comment]
This issue was addressed in 3fef12a, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue no longer occurs as of 3fef12a0b8 for tc012, tc014 (when excluding rows that are NA for all results). Resulting output shown below.

#### tc012

```
> r$FLGACCEPTTAU
 [1] "1" "0" "0" "1" "0" "1" "1" "1" "0" "1" "0" "1" "0" "0" "0" "0" "0" "1" "1" "0" "0" "1" "0" "0" "1" "0" "0" "0" "0"
[30] "1" "1" "1" "1" "1" "1" "1"
```

#### tc014

```
> r[!is.na(r$DOSE1), "FLGACCEPTTAU"]
 [1] "1" "1" "1" "0" "1" "0" "0" "1" "0" "1" "1" "0" "1" "0" "1" "1" "0" "1" "1" "1" "1" "1"
```
